src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,18 +34,23 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={filter.query}
```

Comment:
Ensure that the 'filter.query' is correctly initialized in the Redux state. If 'filter.query' is undefined, it might cause a controlled component error.